src/common/components/testimonial/testimonial.css

```diff
@@ -22,25 +22,22 @@
   }
 }
 
+.pullQuote>p>span {
+  color: #dd4814;
+  font-size: 2rem;
+  line-height: 0;
+  position: relative;
+  top: 0.7rem;
+  left: -0.1rem;
+}
+.pullQuote>p>span + span{
+  left: 5px;
+}
 .pullQuote>p {
   font-size:1em;
   font-style:normal;
-  margin:0
-}
-
-.pullQuote>p:first-of-type:before {
```

Comment:
Yes, I tried keeping them in CSS, but I needed to change their size and couldn't figure out how to do that for just the `:before` and `:after`, so I just stole it from Landscape.

Note that  https://docs.vanillaframework.io/en/base/typography#blockquotes-and-citations uses this `<p>` + `<span>` method